docs/fdroid-submission.md

```
@@ -0,0 +1,405 @@
+```kotlin
+release {
+    …
+    signingConfig = signingConfigs.getByName("release")
+}
+```
+
+On F-Droid's buildserver there is no `jks.properties` and no `KEY_ALIAS`, so the config is empty and
+the assignment still stands. **I verified this empirically rather than reading it off:** I moved
+`jks.properties` aside, unset the four env vars, and ran
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)